main/src/modules/reactiveLayer/behaviorManager/include/pushingRight.h

```diff
@@ -0,0 +1,35 @@
+#ifndef PUSHINGRIGHT
```

Comment:
I don't agree with having millions of behaviors, where each behavior opens several ports. But I will discuss this with @clement-moulin-frier